src/modules/login/login.js

```diff
@@ -253,8 +260,12 @@ export function makeLoginTree (
     stash => stash.appId === appId,
     stash => makeLoginTreeInner(stash, loginKey),
     (stash, children) => {
-      const login = filterObject(stash, ['username', 'appId', 'loginId'])
-      login.children = children
+      const login: LoginTree = filterObject(stash, [
+        'username',
+        'appId',
+        'loginId'
+      ])
+      login.children = children != null ? children : []
```

Comment:
Was this a bug being fixed?

Reply:
No, it's just typing. `children` can't be null in this context, but Flow doesn't know that, so I had to put in the check for the types to work. The fallback case should never be hit.